scripts/exregional_process_bufr.sh

```diff
@@ -183,25 +183,41 @@ cp_vrfy $BUFR_TABLE prepobs_prep.bufrtable
 #   set observation soruce 
 #
 #-----------------------------------------------------------------------
-obs_source=rap
-if [[ ${HH} -eq '00' || ${HH} -eq '12' ]]; then
-  obs_source=rap_e
-fi
+if [[ "${NET}" = "RTMA"* ]]; then
+  SUBH=$(date +%M -d "${START_DATE}")
+  obs_source="rtma_ru"
+  case $MACHINE in
```

Comment:
case in $MACHINE are same for rtma and rrfs. So this piece should not be replicated twice.

Reply:
I will remove the machine dependency in the RTMA if block